bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/FormContainerImpl.java

```diff
@@ -201,18 +197,18 @@ public Map<String, Object> getModel() {
                 }
             }
         } else {
-            FormContainer formContainer = modelFactory.getModelFromWrappedRequest(request, resource, FormContainer.class);
+            FormContainer formContainer = resource.adaptTo(FormContainer.class);
             ObjectMapper mapper = new ObjectMapper();
             jsonMap = mapper.convertValue(formContainer, new TypeReference<Map<String, Object>>() {});
         }
         return jsonMap;
     }
 
     // todo: its similar to other container code, but could not find a better way to do this
-    protected <T> List<T> getChildrenModels(@NotNull SlingHttpServletRequest request, @NotNull Class<T> modelClass) {
+    protected <T> List<T> getChildrenModels(@NotNull Class<T> modelClass) {
         List<T> models = new ArrayList<>();
         for (Resource child : slingModelFilter.filterChildResources(resource.getChildren())) {
-            T model = modelFactory.getModelFromWrappedRequest(request, child, modelClass);
+            T model = child.adaptTo(modelClass);
```

Comment:
Localization would fail. We need to pass request when it is available

Reply:
Can you run this this with your changes, https://github.com/adobe/aem-core-forms-components/pull/125

Reply:
sightly template still works on request object and not in get children models.